src/database/mod.rs

```diff
@@ -52,6 +53,9 @@ pub trait DatabaseConnection {
     /// Update our best chain seen.
     fn update_tip(&mut self, tip: &BlockChainTip);
 
+    /// Get the main descriptor
+    fn main_descriptor(&mut self) -> LianaDescriptor;
```

Comment:
I think you can drop the commit introducing this as it's not used anywhere.

Reply:
Ah yes, this is a leftover from when I created `BdkWallet` using only `db_conn`, but I think it's better to pass the values directly. I'll drop the commit.

Reply:
I've now dropped this commit.